integration-tests/tests/basic-happy-path.spec.ts

```diff
@@ -34,44 +40,91 @@ describe('Create Components using the UI', { tags: ['@PR-check', '@publicRepo']
-      Applications.createdComponentExists(componentNames[1], applicationName);
+    it('Verify we are on "Add Component" wizard, and then hit Cancel', () => {
+      cy.url().should('include', `/import?application=${applicationName}`);
+      addComponent.clickCancel();
+      cy.url().should('include', `${applicationName}/overview`);
     });
   });
 
-  describe('Add a new component using the "Components" tab', () => {
+  describe('Try to add a new component using the "Components" tab', () => {
     it("Use HACBS 'Components' tabs to start adding a new component", () => {
       Applications.goToComponentsTab().clickAddComponent();
     });
 
-    it('Add a component to Application', () => {
-      Applications.createComponent(publicRepos[1], componentNames[2]);
-      Applications.createdComponentExists(componentNames[2], applicationName);
+    it('Verify we are on "Add Component" wizard, and then hit Cancel', () => {
+      cy.url().should('include', `/import?application=${applicationName}`);
+      addComponent.clickCancel();
+      cy.url().should('include', `${applicationName}/components`);
     });
   });
 
-  describe('Add a new component using the "Actions" dropdown', () => {
+  describe('Try to add a new component using the "Actions" dropdown', () => {
     it("Click 'Actions' dropdown to add a component", () => {
       Applications.clickActionsDropdown('Add component');
     });
 
-    it('Add a component to Application', () => {
-      Applications.createComponent(publicRepos[2], componentNames[3]);
-      Applications.createdComponentExists(componentNames[3], applicationName);
+    it('Verify we are on "Add Component" wizard, and then hit Cancel', () => {
+      cy.url().should('include', `/import?application=${applicationName}`);
+      addComponent.clickCancel();
+      cy.url().should('include', `${applicationName}/components`)
+    });
+  });
+
+  describe('Explore Pipeline runs Tab', () => {
+    it("Verify the PipelineRuns List view", () => {
+      Applications.goToPipelinerunsTab();
+
+      cy.get('tbody').find("tr").then((row) => {
+        for (let i = 0; i < row.length; i++) {
+          cy.get('tbody tr').eq(i).then(($row) => {
+            cy.wrap($row).find('td').eq(0).find('a').then((pipelinerunName) => {
+              Applications.createdPipelinerunSucceeded(pipelinerunName.text().trim());
+              pipelinerunsTab.pipelineRunList.push(pipelinerunName.text().trim());
+            });
+          });
+        }
+      });
     });
   });
 
+  describe('Check Component Deployment', () => {
+    it("Verify the status code and response body of the deployment URL of each component", () => {
+      Applications.goToComponentsTab();
+
+      for (let componentName of componentNames) {
+        applicationDetailPage.expandDetails(componentName);
+
+        cy.get(applicationDetailPagePO.route.replace('{0}', componentName), { timeout: 35000 }).invoke('text').then(route => {
+          cy.wait(30000);
```

Comment:
we don't like long unconditional waits here

Reply:
done, thanks